docs/source-pytorch/index.rst

```diff
@@ -62,7 +62,7 @@ Conda users
       </div>
    </div>
 
-Or read the `advanced install guide <starter/installation.html>`_
+Or read the `advanced install guide <https://pytorch-lightning.readthedocs.io/en/stable/starter/installation.html>`_
```

Comment:
```suggestion
Or read the `advanced install guide <installation.rst>`_
```

Reply:
This raises a false positive error, I can do what you have suggested and include it in `linkcheck_ignore`.

Reply:
Resolved. Reset them and added them to `linkcheck_ingore`.